charter.html

```diff
@@ -423,11 +423,20 @@ <h3 id=editors>Editors</h3>
 
 <h3 id=work-mode>Work Mode</h3>
 
-<p>The group will conduct all of its technical work in public. Community
-Group participants agree to make all contributions in the GitHub
-repository the group is using for the particular document. This might be
-in the form of a pull request, by raising an issue, or by adding a
-comment to an existing issue or pull request.
+<p>The group generally conducts its technical work in public.
+
+<p>Community Group participants agree to make all contributions in the
+GitHub repository the group is using for the particular document. This
+might be in the form of a pull request, by raising an issue, or by
+adding a comment to an existing issue or pull request.
+
+<p>This group practices responsible disclosure of security and privacy
+vulnerabilities in our work. When reporting security and privacy
+vulnerabilities in our work, please file an issue in the relevant
```

Comment:
This should also be reflected in a [SECURITY.md](https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) in each repository. That could probably give an email address to contact, or identify a github user with an email address in their profile.

It looks like the privacycg organization can create a default SECURITY.md by [creating a `.github` repository](https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file#creating-a-repository-for-default-files).

Reply:
Thanks, this is a great idea! We've spun up a [.github repo](https://github.com/privacycg/.github/). Would you be willing to suggest text for the SECURITY.md file?

Reply:
No good deed goes unpunished. 😜

I'll send a PR over there. 🙂